src/Observations.jl

```diff
@@ -325,15 +325,15 @@ function initialize_simulation!(simulation, observations, time_series_collector,
     set!(simulation.model, observations, time_index)
 
     times = observation_times(observations)
-
     initial_time = times[time_index]
     simulation.model.clock.time = initial_time
     simulation.model.clock.iteration = 0
     simulation.model.timestepper.previous_Δt = Inf
+    simulation.initialized = false
```

Comment:
what's this?

Reply:
Paranoia. It's set to false when we call `run!`.

Reply:
https://github.com/CliMA/Oceananigans.jl/blob/cf13ab295dc68d74f89fc7dc0792c0588919caef/src/Simulations/run.jl#L94

It makes it so we "initialize" on the first time-step but not otherwise.

Reply:
https://github.com/CliMA/Oceananigans.jl/blob/cf13ab295dc68d74f89fc7dc0792c0588919caef/src/Simulations/run.jl#L157-L193